doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -111,10 +96,9 @@ Programmable proof search
    These flags enable printing of informative or debug information for
    the :tacn:`auto` and :tacn:`trivial` tactics.
 
-.. tacn:: eauto
-   :name: eauto
+.. tacn:: eauto {? @int_or_var } {? @auto_using } {? @hintbases }
```

Comment:
We should also briefly document the new `bfs eauto` tactic.

Reply:
In the time-honored tradition of writers everywhere, I made something up.  Please take a look.